Header header
float64 lat
float64 lon
float64 alt
float64 utm_east
float64 utm_north
uint8 fix
uint8 zone_num
string zone_letter